tests/gaia/ShowNtpCurrent/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,3 @@
+expected_output = {
+    'current': 'The NTP service is inactive'
```

Comment:
maybe below is better? user might want to get just status. 
```
{
    'current_status': 'inactive'
}
```
or output could be vary and not always `The NTP service is XXX`?

Reply:
Agreed - 'inactive' is better. I further reviewed this function and it can actually return 3 possible responses. I've updated the parser to handle all three and have included this comment to `ShowNtpCurrent` to clarify how it now behaves:
```
    # Possible responses to this command are:
    #   'primary and secondary servers are not synchronized'
    #       - This occurs when only a single ntp server is configured, or two are are configured and not synchronized
    #       - return value: {'active': 'unsynchronized'}
    #   'The NTP service is inactive'
    #       - This occurs if the command "set ntp service active on" is missing from the config
    #       - return value: {'active': 'inactive'}
    #   '<ip_address>'
    #       - The IP Address of the NTP server that the clock is currently synchronized to
    #       - return value: {'active': '{ip_address}'}
```

Reply:
not entirely sure, above example says `{'active':`, so you might want to change from `current` to `active` in schema to align with it.

Reply:
Oops - That was a copy/paste error. Corrected the comments to indicate {'current': ...} instead of `active`. That should make it clear that this data is coming from the `show ntp current` command. 

I didn't change the key to "`current_status`" because that only makes sense for the error conditions. When a time server is synchronized and the command returns an ip_address, that's not really a status, that's the currently synchronized clock address. The command itself is somewhat confusing, but I think this implementation is as clear as it's going to get :)